src/components/AppendStepButton.tsx

```diff
@@ -77,14 +83,17 @@ export const AppendStepButton: FunctionComponent<IAddStepButton> = ({
     >
       <Tooltip
         content={ValidationService.getPlusButtonTooltipMsg(showBranchesTab, showStepsTab)}
+        position={layout === 'LR' ? 'top' : 'right'}
       >
         <button
-          className="stepNode__Add plusButton nodrag"
+          className={`${
+            layout === 'LR' ? 'stepNode__Add' : 'stepNode__Add--vertical'
+          } plusButton nodrag`}
```

Comment:
[minor]: I think that in this case, both evaluations will run every time since we're defining those objects here. So what are your thoughts about having something like:

```typescript
// This can be somewhere else or used with useCallback() instead.
const calcClassName = (layout: string): string => {
    const className =  'LR' ? 'stepNode__Add' : 'stepNode__Add--vertical';
    return className + ' plusButton nodrag';
};

const [tooltipPosition, setTooltipPosition] = useState(layout === 'LR' ? 'top' : 'right');
const [buttonClassName, setButtonClassName] = useState(calcClassName(layout));

useEffect(() => {
    setTooltipPosition(layout === 'LR' ? 'top' : 'right');
    setButtonClassName(calcClassName(layout));
}, [layout]);
```

and then lastly just bind the variables in the template part

```jsx
<Tooltip
  content={ValidationService.getPlusButtonTooltipMsg(showBranchesTab, showStepsTab)}
  position={tooltipPosition}
  >
        <button
          className={buttonClassName}
          ...
        />
</Tooltip>
```